docs/guides/roles-responsibilities.md

```diff
@@ -0,0 +1,58 @@
+# Roles & Responsibilities
+
+Shipping web software requires the effort of a lot of people. While each client and project is different, the general shape of a project team looks something like this:
+
+
+## Stakeholder
+
+Blanket term that usually encompasses various client stakeholders (ex: marketing, operations, support, dev team, etc.) and end customers (users of the client’s product).
```

Comment:
Noticed we dropped a paragraph from the Stakeholder description; was that intentional? https://docs.v1.thinktandem.io/project-management/roles-responsibilities.html#stakeholder

Reply:
Yep, I intentionally dropped that paragraph because it didn't *seem* to be something that we were doing in the projects I had been a part of looking at the brief for. I'd definitely be down for adding it back in with language like: "We strive to do x when possible because of benefit y" which would reflect it as a goal that we don't always accomplish due to project specifics.